native/KotlinIOS/KotlinIOS/Base.lproj/Main.storyboard

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
From Xcode it looks like you've used the autoresizing masks instead of adding constraints to position your elements. That's totally reasonable, but generally on the LNER project we use explicit constraints so it would probably be good to get into that habit now